src/pkg/projects/netcoreapp/sfx/theme/1046/bundle.wxl

```diff
@@ -58,8 +58,8 @@ Preparado? Tudo definido? Vamos nessa!</String>
   <String Id="FilesInUseDontCloseRadioButton">&amp;Não feche os aplicativos. Será necessária uma reinicialização.</String>
   <String Id="FilesInUseOkButton">&amp;OK</String>
   <String Id="FilesInUseCancelButton">&amp;Cancelar</String>
-  <String Id="WelcomeHeaderMessage">Tempo de Execução do .NET Core</String>
-  <String Id="WelcomeDescription">O .NET Core é uma plataforma de desenvolvimento que você pode usar para criar aplicativos de linha de comando, microsserviços e sites modernos. É software livre, multiplataforma e com suporte da Microsoft. Esperamos que você goste!</String>
+  <String Id="WelcomeHeaderMessage">Runtime do .NET Core</String>
+  <String Id="WelcomeDescription">O .NET Core é uma plataforma de desenvolvimento que você pode usar para criar aplicativos de linha de comando, microsserviços e sites modernos. Trata-se de um software livre, multiplataforma que conta com o suporte da Microsoft. Esperamos que você goste dele!</String>
```

Comment:
Are the changes in this line and `Motto` expected? Are they just some extra improvements we might as well include while we're changing the file?

In one place for easy review:

```diff
-Preparado? Tudo definido? Vamos nessa!</String>
+Tudo pronto? Então, vamos nessa!</String>

-  ...modernos. É software livre, multiplataforma e com suporte da Microsoft. Esperamos que você goste!</String>
+  ...modernos. Trata-se de um software livre, multiplataforma que conta com o suporte da Microsoft. Esperamos que você goste dele!</String>
```

Reply:
I believe it is improvement of translation. @janbaltus, please confirm. Thanks!

Reply:
Yes this is translation impprovement after LQA